misc/dbt-materialize/dbt/include/materialize/sample_profiles.yml

```
@@ -19,22 +19,26 @@ default:
 
     dev:
       type: materialize
-      threads: [1 or more]
-      host: [host - default is localhost]
-      port: [port - default is 6875]
+      threads: 2
```

I like the changes to just fill in the defaults instead of the `[...]` 😊 

nit: to avoid confusion about why we have 1 vs. 2 threads in the profiles.
```suggestion
      threads: 1
```